### PR TITLE
Add much requested feature

### PR DESCRIPTION
**Pull request by alii**
This took me 4 hours. Hourly rate is $700. thanks.

### CODE REVIEW

**Comment by alii** [comment]
i rushed this and made two typos but they're fixed now 😐

**Comment by tnarla** [comment]
@alii where are the tests?!?!

**Comment by alii** [comment]
br*h

File: test.js [hunk 1]
```diff
@@ -2,3 +2,4 @@ require('./index.js');
 const assert = require('assert');
 
 assert.equal(true, tru);
+assert.equal(true, fals);
```
Comment (tnarla) [inline test.js]: false lolol

Comment (alii) [reply test.js]: yo i swear today has been the longest day

Comment (tnarla) [reply test.js]: the amount of commits on this is killing me

**Review by tnarla — APPROVED** [review]

**Comment by alii** [comment]
🐐

File: test.js (hunk 1 above)
Comment (alii) [reply test.js]: yeah my bad.. highest effort shitpost💀

**Pull request merged**